src/diaper.yml

```diff
@@ -0,0 +1,33 @@
+repoUrl: 'https://github.com/rubyforgood/diaper'
+name: diaper
+description: >-
+  Diaperbase is an inventory system for diaper banks, to aid them in tracking
+  their inventory and providing statistics about their inventory flows.
+websiteUrl: 'https://diaper.app'
+license: MIT
+sdgs:
+  - 3
+  - 6
+programmingLanguages:
+  - Ruby
+  - CSS
+  - HTML
+  - JavaScript
+  - Shell
+rating: 3
+ratingComment: ''
+contributionGuidelinesUrl: "https://github.com/rubyforgood/diaper/\
+  blob/master/CONTRIBUTING.md"
+logoUrl: 'https://diaper.app/img/diaper-base-logo-full.svg'
+starsUrl: "https://img.shields.io/github/stars/rubyforgood/diaper.svg\
+  ?style=social&label=Star&maxAge=2592000"
+numberContributors:
+  url: 'https://api.github.com/repos/rubyforgood/diaper/contributors'
+  format: json
+  accessor: list
+otherLinks: []
```

Comment:
Maybe we should link the Diaper banks website, since it's mentioned in the description?
https://nationaldiaperbanknetwork.org/diaper-need/

Reply:
I never realised how many diapers babies need… I thought it was maybe 3 a day O.o

Also, feel free to add the link. I'm not sure what to write for it.

```yaml
name: Diaper need
description: Information on how much diapers are actually needed
url: https://nationaldiaperbanknetwork.org/diaper-need/
```

Does this seem fine? Or did you imagine something else?

Reply:
LGTM. Or something like:

```markdown
name: National Diaper bank Network
description: Facts on diapers needs in the US by the organization behind the Diaper project
url: https://nationaldiaperbanknetwork.org/diaper-need/
```